app/helpers/Emails/EmailHelper/email_cs.latte

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>{$subject}</title>
+  <style>
+    a {
+      font-weight: bold;
+      color: #595959;
+      border-bottom: 1px solid #cfcfcf;
+      text-decoration: none;
+    }
+
+    hr {
+      color: #cbd9dc;
+      background-color: #cbd9dc;
+    }
+
+    p {
+      color: #000;
+    }
+  </style>
+</head>
+<body style="margin: 0;">
+  <div style="padding: 30px 0; font-family: Arial, sans-serif; font-size: 10pt;">
+    <table style="margin: 0 auto;" align="center">
+      <tr>
+        <td style="padding: 0 !important;">
+          <table style="width: 600px; margin: 0 auto; background: #fff; border: 1px solid #cbd9dc;" align="center" cellspacing="0" cellpadding="0">
+            <tr>
+              <td style="background: #222D32; color: #ecf0f5; font-weight: bold; padding: 20px;">
+                <h2 style="margin: 0;">{$subject}</h2>
+              </td>
+              <td style="background: #222D32; margin: 0; padding: 10px 20px 10px 0;text-align: right;">
+                <img style="width: 50px; border: none;" src="{$apiUrl|trim:"/"}/emails/img/logo_only.png" alt="ReCodEx">
+              </td>
+            </tr>
+          
+            <tr>
+              <td style="padding: 20px;" colspan="2">
+                <div style="word-wrap: break-word; width: 580px;">
+                  {$message|noescape}
+                </div>
+                <hr>
+                <p>
+                  Pokud nechcete dostávat tyto emaily, aktualizujte prosím svoje uživatelské nastavení.
+                </p>
+              </td>
+            </tr>
+            <tr style="background: #f8f8f8; margin: 0; padding: 0; text-align: left; overflow: hidden;">
+              <td style="margin: 0; padding: 20px;">
+                <p style="margin: 0; padding: 0;">
+                  Prgejte zodpovědně,<br>ReCodEx Team<br>
+                  <a href="{$footerUrl}">{$footerUrl}</a>
+                </p>
+              </td>
+              <td style="margin: 0; padding: 20px; text-align: right;">
+                <a href="{$githubUrl}" style="width: 30px; height: 30px; padding-left: 10px; border: none;" target=_blank>
+                  <img style="border: none;" src="{$apiUrl|trim:"/"}/emails/img/github.png" width="30" height="30" alt="Github">
+                </a>
+              </td>
+            </tr>
+          </table>
+        </td>
+      </tr>
+    </table>
+  </div>
+</body>
+</html>
```

Comment:
If I understand correctly, the structure of this template is a copy of English template. Hence, if we need to change the appearance, we need to do it correctly in two places. Since the template holds only a few messages, I believe that there should be only one template, and localization should be handled outside of the template.

Reply:
I agree, but I would like to resolve this the same way as mail prefixes, but that is entirely new functionality and deserves its own PR.